pyabc.py

```diff
@@ -293,6 +293,7 @@ def __sub__(self, x):
 
 class TimeSignature(object):
     def __init__(self, meter, unit_len, tempo=None):
+        meter = meter.replace('C|', '4/4').replace('C', '4/4')
```

Comment:
This is the only real change. I seem to have been a little careless and one of my editors has must have some sort of auto linting, all the other changes are simply removing whitespace.

Reply:
`C|` should be cut time `2/2`, correct?
http://abcnotation.com/wiki/abc:standard:v2.1#mmeter

Reply:
I think you are, I'll correct this.